### PR TITLE
Add WebSocket support for log streaming

### LINKED ISSUE #52: Better log streaming solution
When streaming logs via `curl`, it seems to disconnect after a minute or so. I'm not sure exactly where this issue originates (Docker API/Gin/nginx), but I'll find out 🔎

This is the error:
```
curl: (92) HTTP/2 stream 0 was not closed cleanly: INTERNAL_ERROR (err 2)
```

### PR DESCRIPTION
**Pull request by cjdenio**
Resolves #52

### CODE REVIEW

File: pkg/api/apps/logs.go [hunk 1]
```diff
@@ -89,7 +101,10 @@ func handleGETLogs(c *gin.Context) {
 
 		for scanner.Scan() {
 			log_mutex.Lock()
-			log_chan <- append([]byte("[stdout] "), scanner.Bytes()...)
+			log_chan <- map[string]interface{}{
```
Comment (anirudhb) [inline pkg/api/apps/logs.go]: `gin.H` works here too

File: pkg/api/apps/logs.go [hunk 2]
```diff
@@ -99,28 +114,35 @@ func handleGETLogs(c *gin.Context) {
 
 		for scanner.Scan() {
 			log_mutex.Lock()
-			log_chan <- append([]byte("[stderr] "), scanner.Bytes()...)
+			log_chan <- map[string]interface{}{
```
Comment (anirudhb) [inline pkg/api/apps/logs.go]: Ditto

**Review by anirudhb — APPROVED** [review]
lgtm

File: pkg/api/apps/logs.go (hunk 1 above)
Comment (cjdenio) [reply pkg/api/apps/logs.go]: interesting, thanks!

**Pull request merged**